torch/csrc/utils/pycfunction_helpers.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#include <Python.h>
+#include <torch/csrc/WindowsTorchApiMacro.h>
+
+PyCFunction convertPyCFunctionWithKeywords(PyCFunctionWithKeywords func);
```

Comment:
I'd suggest to implement this as an inline function here to avoid the overhead at startup time. This is a simple cast and hence can be completely eliminated if the compiler can see it.
Nit: Rename to `cast...` in reference of `static_cast/reinterpret_cast` which is what this essentially does.